h/static/styles/app.scss

```diff
@@ -117,6 +118,9 @@ $group-list-width: 270px;
 
   .dropdown-menu {
     width: $group-list-width;
+    max-height: 500px;  // fallback for browsers lacking support for vh/calc
```

Comment:
Do you want me to remove this? Seems like a pretty harmless fallback in any case.